<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Power</class>
 <widget class="QDialog" name="X2Power">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>574</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>496</width>
    <height>574</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>496</width>
    <height>574</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pegasus Pocket Powerbox Advance settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>99999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_logo_2">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>-8</y>
        <width>256</width>
        <height>90</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="pixmap">
       <pixmap>PegasusAstro.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>PegasusAstro.png</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>488</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>376</x>
        <y>488</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>88</y>
        <width>248</width>
        <height>160</height>
       </rect>
      </property>
      <property name="title">
       <string>Dew Ports</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>32</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew Heater A :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>64</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew Heater B :</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="dewHeaterA">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>32</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="dewHeaterB">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>64</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>32</y>
         <width>50</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>64</y>
         <width>50</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>96</y>
         <width>87</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto Dew</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>128</y>
         <width>112</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Aggressiveness :</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>128</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>192</y>
        <width>176</width>
        <height>56</height>
       </rect>
      </property>
      <property name="title">
       <string>LED Indicator</string>
      </property>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>24</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>On</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>24</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>88</y>
        <width>176</width>
        <height>88</height>
       </rect>
      </property>
      <property name="title">
       <string>Adjustable power port</string>
      </property>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>40</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Power  :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>256</y>
        <width>440</width>
        <height>184</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>32</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Voltage :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>32</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Temp :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>56</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Humidity :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>272</x>
         <y>56</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew Point :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="voltage">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>32</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- V</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="temperature">
       <property name="geometry">
        <rect>
         <x>368</x>
         <y>32</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="humidity">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>56</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>- %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="dewPoint">
       <property name="geometry">
        <rect>
         <x>368</x>
         <y>56</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>-8</x>
         <y>80</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Average  Current draw :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentDraw">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>128</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Amp Hours :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="ampHours">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>128</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- Ah</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>152</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Watt Hours :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="wattHours">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>152</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- Wh</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="totalCurrentDraw">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>104</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>-8</x>
         <y>104</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Current draw :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>80</y>
         <width>136</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>12 V Current draw :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentDraw12V">
       <property name="geometry">
        <rect>
         <x>376</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentDrawDewA">
       <property name="geometry">
        <rect>
         <x>376</x>
         <y>104</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>104</y>
         <width>136</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew A Current draw :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentDrawDewB">
       <property name="geometry">
        <rect>
         <x>376</x>
         <y>128</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>128</y>
         <width>136</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew B Current draw :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>448</y>
        <width>248</width>
        <height>88</height>
       </rect>
      </property>
      <property name="title">
       <string>Ports boot state</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>48</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string> State on boot :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>144</x>
         <y>48</y>
         <width>16</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>208</x>
         <y>48</y>
         <width>16</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>40</y>
         <width>3</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>24</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>4 x 12V</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>24</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Adjust.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
